parm/config/gefs/config.fcst

```diff
@@ -148,6 +151,9 @@ if (( LEVS == 128 )) && [[ "${CDUMP}" =~ "gdas" ]]; then
    export d2_bg_k2=0.0
 fi
 
+# n_split adjustment
+export n_split=5
+
```

Comment:
Yes, this is the case.  We had gone from 'n_split=6' to 'n_split=4' (larger dynamics time step for faster runs) and started experiencing numerical instability crashes at C768.  Compromising with 'n_split=5' eliminated the crashes.

Reply:
Ideally, we would have `n_split` based on resolution as part of `config.ufs`.  But since we are prescribing the same for all resolutions, I am inclined to suggest updating the default on line 384 in `ush/forecast_postdet.sh`

Reply:
I updated the default in ush/forecast_postdet.sh and removed the explicit setting in config.fcst.